src/lib/y2network/widgets/routing_table.rb

```diff
@@ -23,7 +23,7 @@
 require "y2network/interface"
 
 Yast.import "Label"
-Yast.import "NetworkService"
+Yast.import "Lan"
```

Comment:
it seems that import of the Lan module is getting everywhere ... so we probably could think of separating new backend "accessor" from the Lan module

Reply:
The Lan is currently the module which stores the configs. If we add a config manager or something like that, where we could retrieve from the different configs, then, we could get rid of it. But currently it is our central store.